CHANGELOG.md

```diff
@@ -5,11 +5,10 @@ All notable changes to PowerModelsMCDC.jl will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-
-### Fixed
```

Comment:
It is [recommended](https://keepachangelog.com/en/1.1.0/#how) to keep the categorization of changes into < added | changed | deprecated | removed | fixed | security >.